### ISSUE TITLE
Connect should determine possible DisplayRotation values, not Suite

### ISSUE THREAD

**Issue by komret** [trezor/trezor-suite#13360]
DisplayRotation values are currently determined by Suite, but Connect would be a more logical place so that this property is available to third-party apps. See discussion [here](https://github.com/trezor/trezor-firmware/issues/3966). Connect should provide an array with possible values, i.e. `[0, 90, 180, 270]` for T2T1 and T3T1, `[0, 180]` for T2B1.

**Comment by mroz22** [comment]
how to do it:
1. create a new types patch for `Features.display_rotation` not sure if we are ready to introduce a new Enum? 
2. profit

**Comment by mroz22** [comment]
btw why won't trezor-firmware define `Features.display_rotation` as Enum? Similar to `HomescreenFormat` for example. cc @matejcik is that out of the question?

**Comment by matejcik** [comment]
currently allowed values are plain ints, correct? then sure, we can define an enum for them

**Comment by mroz22** [comment]
exactly, now it is ints and since only specific ints (0, 90,..) are allowed, it would make sense to define them as enums. this would simplify code on our side a bit. may we request this, please? Or maybe @Hannsek to put it somewhere into the backlog as a "first good issue"

**Comment by Hannsek** [comment]
So define an enum for the possible values and put them into `get_features`?
